clientconn.go

```diff
@@ -402,6 +404,8 @@ type ClientConn struct {
 	balancerWrapper *ccBalancerWrapper
 	retryThrottler  atomic.Value
 
+	resolvedOnce *grpcsync.Event
```

Comment:
The name... This looks like a `sync.Once`...
How about `resolvedEvent`?

Reply:
`firstResolveEvent`?